src/vs/workbench/contrib/interactiveSession/browser/actions/interactiveSessionCopyActions.ts

```diff
@@ -0,0 +1,79 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ServicesAccessor } from 'vs/editor/browser/editorExtensions';
+import { localize } from 'vs/nls';
+import { Action2, MenuId, registerAction2 } from 'vs/platform/actions/common/actions';
+import { IClipboardService } from 'vs/platform/clipboard/common/clipboardService';
+import { INTERACTIVE_SESSION_CATEGORY } from 'vs/workbench/contrib/interactiveSession/browser/actions/interactiveSessionActions';
+import { IInteractiveSessionWidgetService } from 'vs/workbench/contrib/interactiveSession/browser/interactiveSessionWidget';
+import { IInteractiveRequestViewModel, IInteractiveResponseViewModel, isRequestVM, isResponseVM } from 'vs/workbench/contrib/interactiveSession/common/interactiveSessionViewModel';
+
+export function registerInteractiveSessionCopyActions() {
+	registerAction2(class CopyAllAction extends Action2 {
+		constructor() {
+			super({
+				id: 'workbench.action.interactiveSession.copyAll',
+				title: {
+					value: localize('interactive.copyAll.label', "Copy All"),
+					original: 'Copy All'
+				},
+				f1: true,
```

Comment:
This is inside the interactive sessions, so it shouldn't be in the command palette.